clients/src/test/java/org/apache/kafka/clients/consumer/internals/CommitRequestManagerTest.java

```diff
@@ -922,7 +922,11 @@ public void testEnsureBackoffRetryOnOffsetCommitRequestTimeout() {
         assertRetryBackOff(commitRequestManager, retryBackoffMs);
     }
 
-    private void assertCoordinatorDisconnect() {
+    private void assertCoordinatorDisconnectOnDisconnect() {
```

Comment:
names got a bit confusing with the other func I would say. Would it be clearer if we rename this one to show that it's about assertCoordinatorDisconnectHandling maybe? (it may or may not be a coordinator disconnect, depending on the specific error, but we ensure it's being handled)

Reply:
I think `assertCoordinatorDisconnectHandling` is a good one, thanks 😺